Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py

```diff
@@ -284,7 +289,12 @@ def main():
         else:
             raise NotImplementedError(f'{command} is not an existing Generic SQL command')
     except Exception as err:
-        return_error(f'Unexpected error: {str(err)} \nquery: {demisto.args().get("query")} \n{traceback.format_exc()}')
+        if type(err).__name__ == 'OperationalError':
```

Comment:
how do you know this is the only case that you will get error named "OperationalError" seems to me that this is can be a family of different errors.
maybe it is better to check if e substring in the error message matches? like if you have 'certificate' or something like this?(need to check the original error.

in any case, you should print the error content in the end and not to hide it.